docs/index.rst

```diff
@@ -6,8 +6,223 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with a python package with two directories inside:
+
+- metadata folder with :code:`yml` files holding the list of errors code can trigger
+- checks folder with python :code:`check_` functions that actually perform the checks
+
+Both of them need to be *accessible to the zoo* and the metadata and checks filenames have to match (the zoo imports the metadata files based on the python check filename).
```

Comment:
I'd change `accessible to` for `importable by`. This can help clarify the package specification.